bot/voice.go

```diff
@@ -0,0 +1,99 @@
+package bot
+
+import (
+	"log"
+	"strings"
+	"time"
+
+	"github.com/bwmarrin/discordgo"
+)
+
+var activeReceiver *AudioReceiver
+
+func stopActiveReceiver() {
+	if activeReceiver != nil {
+		activeReceiver.Stop()
+		activeReceiver = nil
+	}
+}
+
+func dcCmd(ctx CommandCtx) error {
+	if ctx.Message.Author.ID == morthisId {
+		var uId string
+
+		if len(ctx.Args) < 1 {
+			uId = venovaId
+		} else {
+			uId = strings.Trim(ctx.Args[0], "<@>")
+		}
+
+		if uId == venovaId {
+			stopActiveReceiver()
+		}
+		err := disconnectUserFromVC(ctx.Session, ctx.Message.GuildID, uId)
+		if err != nil {
+			return ctx.Reply("Error disconnecting from voice channel.")
+		}
+		if uId == venovaId {
+			return ctx.Reply("Disconnected Venova from the server.")
+		} else {
+			return ctx.Reply("Disconnected user from voice channel.")
+		}
+	}
+	return ctx.Reply("stinky")
+}
+
+func disconnectUserFromVC(session *discordgo.Session, guildID, userID string) error {
+	err := session.GuildMemberMove(guildID, userID, nil)
+	if err != nil {
+		log.Printf("Failed to disconnect user %s from VC: %v", userID, err)
+		return err
+	}
+
+	log.Printf("User %s has been disconnected from voice.", userID)
+	return nil
+}
+
+func monitorVoiceActivity(session *discordgo.Session, guildID, channelID string) {
+	ticker := time.NewTicker(1 * time.Minute)
+
+	for range ticker.C {
+		guild, err := session.State.Guild(guildID)
+		if err != nil {
+			log.Printf("Error fetching guild: %v", err)
+			continue
+		}
+
+		userCount := 0
+		for _, vs := range guild.VoiceStates {
+			if vs.ChannelID == channelID {
+				userCount++
+			}
+		}
+
+		if userCount == 1 {
```

Comment:
Another case where we can greatly simplify control-flow and readability by inverting an if:

```go
if userCount != 1 {
    continue
}

...
return
```